examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -368,6 +370,62 @@ angular.module('cloudberry.common', ['cloudberry.mapresultcache'])
             break;
             
           case 'heatmap':
+            var heatJson = (JSON.stringify({
+              dataset: parameters.dataset,
+              filter: getFilter(parameters, defaultHeatmapSamplingDayRange, parameters.geoIds),
+              select: {
+                order: ["-create_at"],
+                limit: defaultHeatmapLimit,
+                offset: 0,
+                field: ["id", "coordinate", "place.bounding_box", "create_at", "user.id"]
+              },
+              option: {
+                sliceMillis: cloudberryConfig.querySliceMills
+              },
+              transform: {
+                wrap: {
+                  id: "heat",
+                  category: "heat"
+                }
+              }
+            }));
+
+            // for the time histogram
```

Comment:
Why can't we use the same query from CountMap or PinMap?

Reply:
Agree. It is more like a place holder for now.

Reply:
OK